doc/api/fs.md

```diff
@@ -2867,6 +2883,8 @@ changes:
 
 Synchronous symlink(2). Returns `undefined`.
 
+For detailed information, see the documentation of [`fs.symlink()`][].
```

Comment:
It seems we miss the bottom reference for this link. You can add this to the bottom list (in ASCII sorting order):
```md
[`fs.symlink()`]: #fs_fs_symlink_target_path_type_callback
```